pkg/services/tree/sync.go

```diff
@@ -19,7 +19,7 @@ import (
 var ErrNotInContainer = errors.New("node is not in container")
 
 // Synchronize tries to synchronize log starting from the last stored height.
-func (s *Service) Synchronize(ctx context.Context, cid cid.ID, treeID string) error {
+func (s *Service) Synchronize(ctx context.Context, cid cid.ID, treeID *string) error {
```

Comment:
Upd docs?

Reply:
was changed according to another comment